src/workerd/api/node/dns.h

```diff
@@ -117,11 +160,15 @@ class DnsUtil final: public jsg::Object {
     JSG_METHOD(setDefaultResultOrder);
     JSG_METHOD(getDefaultResultOrder);
     JSG_METHOD(setServers);
+
+    // Classes
+    JSG_NESTED_TYPE_NAMED(DnsResolver, Resolver);
   }
 };
 
 #define EW_NODE_DNS_ISOLATE_TYPES                                                                  \
-  api::node::DnsUtil, api::node::DnsUtil::LookupOptions, api::node::DnsUtil::Resolve4Options,      \
-      api::node::DnsUtil::Resolve6Options
+  api::node::DnsUtil, api::node::DnsUtil::LookupOptions,                                           \
```

Comment:
will this remove `Resolve4Options` from api? Is it a breaking change? it doesn't seem so, just trying to be careful here.

Reply:
I just renamed it to ResolveOptions. No breaking changes. This API is not implemented yet. All functions have a stub implementation.